sql/fishing_contest.sql

```diff
@@ -0,0 +1,13 @@
+--
+-- Table structure for table `fishing_contest`
+-- This table holds the parameters for the current fish ranking contest
+--
+
+CREATE TABLE IF NOT EXISTS `fishing_contest` (
```

Comment:
Is this going to be a single entry at all times, or does the entire history forever need to be retained (I have no idea how fishing or contests work)

Reply:
For now, it's a single entry.  I built the original of this to capture all history, but took that out since it's not really something that retail does.  With this method, there should only be a single entry in the table, ever.  I could do this as a server var as well, but this made it easier to have the data not buried by everything else.

Reply:
I think it's fine to have the table with a single row, we do something very similar with conquest I think, and will probably do the same in the future with besieged - no need to stuff everything into server vars